### ISSUE TITLE
Progress bar example updates

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1128]
Some updates to our progress bar examples are needed to better align with our design documentation and also provide better clarity regarding how to utilize the progress bar for different use cases. 

- The text string "Title" should display as the primary text
  - For most examples, "Title" would replace "Descriptive text here"
  - For the success and failure examples, "Title" would replace the strings "Success" and "Failure due to an error"
- The example labeled "Progress with Dynamic Description" can support a couple of use cases. To more clearly represent these use cases in our examples, we should create two versions of this example showing the following use cases:
  1. progress value is numeric but not a percentage, example 1
      - In this case, the value displays on the right where the percentage value currently displays, but in the format "2 of 5 units"
      - use `aria-valuetext="2 of 5 units"` in this example
      - other attributes can be set to `aria-valuemin="0" aria-valuemax="5" aria-valuenow="2"`
  2. progress value is numeric but not a percentage, example 2
      - In this case, the value displays on the right where the percentage value currently displays, but as a text string that describes the current value. 
      - The text string for the example we show could say "Step 2: Copying files"
      - use `aria-valuetext="Step 2: Copying files"`
      - other attributes should still be included if they can accurately reflect current progress, e.g.: `aria-valuenow="2" aria-valuemin="0" aria-valuemax="4"`

- The following documentation about `aria-valuetext` should be included:
    > Provide a text string that communicates current status. Only use if the important information about status is included in the text string. Do no use if percentage is the most important value to communicate. Some screen readers will ignore the percentage value determined from `aria-valuenow` when `aria-valuetext` is used.

**Comment by jgiardino** [comment]
Notes about aria attributes related to `role="progressbar"`

- JAWS will ignore the following:
    - `aria-valuemin`
    - `aria-valuemax`
    - `aria-valuenow` IF `aria-valuetext` is included
- VO will announce all of these as expected

Here are some recommendations for how to use these attributes for different cases:

1. If `aria-valuetext` is not being used, then given the way JAWS handles these attributes, they should always have these values:
    - `aria-valuemin="0"`
    - `aria-valuemax="100"`
    - `aria-valuenow` should only be used to indicate percentage
2. If `aria-valuetext` is needed to include additional status text and percentage is important to include, then:
    - Include percentage as part of `aria-valuetext`
    - Do not include `aria-valuenow` (to avoid duplicate information announced by VO)
3. If `aria-valuetext` is needed to include additional status text, and percentage is not critical to be announced, then all attributes can be used as specified. For example, with these attributes:
    - `aria-valuemin="0" aria-valuemax="5" aria-valuenow="2" aria-valuetext="Step 2: Copying files"`
    - JAWS will announce "Step 2 Copying files, progress bar"
    - VO will announce "Step 2 Copying files, 20 percent, progress indicator"
